ckanext/dcat/profiles.py

```diff
@@ -991,15 +993,21 @@ def parse_dataset(self, dataset_dict, dataset_ref):
 
         # Basic fields
         for key, predicate in (
-                ('title', DCT.title),
-                ('notes', DCT.description),
-                ('url', DCAT.landingPage),
```

Comment:
The `parse_dataset()` method essentially maps the RDF graph from the remote semantic serialization to a CKAN dataset dict. If we remove these lines we are not extracting the remote values anymore. If we just do 

     dataset_dict['title'] = toolkit.h.get_translated(dataset_dict, 'title')

I don't see how we can actually obtain the remote value for `title`. And besides, IIUC  the only thing `h.get_translated()` does is looking for an existing `title_translated[lang]` field in the dataset dict, which at this point won't exist.

Without knowing what you are trying to achieve in detail, I guess if the final goal is to create a dataset dict with fluent-like metadata fields, we would need to
1. Parse whatever way DCAT/RDF has of defining multilingual metadata
2. Use that information to create the relevant `*_translated` fields

This would only work if the user is using ckanext-fluent, so perhaps that should be an opt-in behaviour

Reply:
@amercader okay, so will I need to even do anything in the `profiles.py` even? Is everything related to what I need to do this fluent stuff just in the `converters.py`?

Reply:
N/A `profiles.py` reverted.